pkgs/top-level/all-packages.nix

```diff
@@ -5953,8 +5951,6 @@ in
 
   jump = callPackage ../tools/system/jump {};
 
-  kindlegen = callPackage ../tools/typesetting/kindlegen { };
```

Comment:
Can you add kindlegen to `pkgs/top-level/aliases.nix`.

Reply:
Done!